proposed/2021/TargetFrameworkAndPlatformsFilters.md

```diff
@@ -0,0 +1,196 @@
+# Target Framework and Platform Filters
+
+- [Jon Douglas](https://github.com/JonDouglas)
+- Start Date (2021-11-09)
+- [Package discovery: Show and search by TFM/package-compatibility #3098](https://github.com/NuGet/NuGetGallery/issues/3098)
+
+## Summary
+
+<!-- One-paragraph description of the proposal. -->
+When a developer is browsing for a NuGet package to include in their project, they can filter packages based on generations of .NET including the target framework & supported platforms that package targets.
+
+## Motivation
+
+<!-- Why are we doing this? What pain points does this solve? What is the expected outcome? -->
+Finding a package that is compatible with your project is a daily challenge for the average developer in the .NET ecosystem. Over 21% of developers fail to install a package today because of an incompatibility; it's one of NuGet's top errors known as `NU1202`. Ontop of it being one of the biggest error codes we see, it is also [the number #1 request on the `NuGetGallery` repository](https://github.com/NuGet/NuGetGallery/issues/3098) for good reason. It should be straight-forward for a developer to separate the generation of .NET they are currently targeting and find packages that are supported on the platforms that the respective generation supports.
+
+## Explanation
+
+### Functional explanation
+
+<!-- Explain the proposal as if it were already implemented and you're teaching it to another person. -->
+<!-- Introduce new concepts, functional designs with real life examples, and low-fidelity mockups or  pseudocode to show how this proposal would look. -->
+When browsing for packages on NuGet.org, a developer will be able to filter packages based on compatibility filters. These filters will include:
+
+1. The **generation of .NET** such as .NET Framework, .NET Core, .NET (5, 6, and later), and Any.
```

Comment:
Customers may be really confused by .NET and .NET Core. Support for the last .NET Core TFM will be dropped in December 2022: https://dotnet.microsoft.com/platform/support/policy/dotnet-core#lifecycle

Should we avoid the confusion and "merge" .NET and .NET Core?

Reply:
Will need to get more feedback from others regarding how .NET Core & .NET 5+ are perceived. .NET 5.0 is still technically `.NETCoreApp,Version=v5.0` for example.

Reply:
Agreed. The proper name is ideal for this scenario. We will stay with them to keep them consistent with other badges and similar.